rs/nns/governance/src/neuron_store.rs

```diff
@@ -105,11 +105,6 @@ impl NeuronStoreError {
     }
 }
 
-#[derive(Debug, Eq, PartialEq)]
-pub struct NeuronNotFound {
-    neuron_id: NeuronId,
-}
-
```

Comment:
Is this causing a warning or some other problem? Otherwise, please, revert.

Reply:
It is dead code :)

Clippy rightly gives a warning making CI fail.

Reply:
Sure. But I have received comments going the other way, where I commented out dead code or added a `allow` statement, but was requested to delete it.

If the code is unused I will assume it dead code unless someone points it out during a code review.